zkevm-circuits/src/evm_circuit.rs

```diff
@@ -460,15 +460,15 @@ pub mod test {
         .unwrap();
 
         let k = log2_ceil(NUM_BLINDING_ROWS + rows_needed);
-        dbg!([
-            num_rows_required_for_rw_table,
-            num_rows_required_for_fixed_table,
-            num_rows_required_for_bytecode_table,
-            num_rows_required_for_copy_table,
-            num_rows_required_for_keccak_table,
-            num_rows_required_for_tx_table,
-            num_rows_required_for_exp_table
-        ]);
+        // dbg!([
+        //     num_rows_required_for_rw_table,
+        //     num_rows_required_for_fixed_table,
+        //     num_rows_required_for_bytecode_table,
+        //     num_rows_required_for_copy_table,
+        //     num_rows_required_for_keccak_table,
+        //     num_rows_required_for_tx_table,
+        //     num_rows_required_for_exp_table
+        // ]);
```

Comment:
Ah I didn't meant to leave this commented, I just wanted to do it locally to avoid too much noise during tests.  I've updated it to `log::debug` following your suggestion as I think it's cleaner :)  Done in https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1046/commits/f59933089f977ec564f751c7051a824c0d5dd041